src/chainparams.cpp

```diff
@@ -261,10 +261,10 @@ class CTestNetParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 0000000000004ae2f3896ca8ecd41c460a35bf6184e145d91558cece1c688a76
-            /* nTime    */ 1625727096,
-            /* nTxCount */ 60408943,
-            /* dTxRate  */ 0.08379062270367649,
+            // Data from RPC: getchaintxstats 4096 00000000d18cfe81cbeea665076807789bd8f831d557632e635bc6e3c003069e
```

Comment:
Why not use the assumed valid block for this?

Reply:
Sure, why not, I don't know. If you have other requirements, please update the release process. I'm done with this for this release.

Reply:
It's probably easier for new reviewers, we can indeed do that for that for the next release.

Reply:
Nice.

Reply:
"Testnet should be set with a height some tens of thousands back from the tip, due to reorgs there" (release-process.md) -> would that apply only for updating `nMinimumChainWork` then?